lms/services/mailchimp.py

```diff
@@ -0,0 +1,71 @@
+import logging
+from dataclasses import dataclass
+
+import mailchimp_transactional
+
+LOG = logging.getLogger(__name__)
+
+
+@dataclass
+class EmailSender:
+    subaccount: str
```

Comment:
Where would this value come from? I'd assume it will be from an envar for 100% of the email digests?

I guess you are trying to abstract it here as a service for every email we send and read the envar from the "digest" focused service? fair enough if that's the case.

Reply:
Yep, it'll be an envvar. We might want to use different subaccounts for different types of email. But I think it'll always be the one subaccount at least for all instructor daily digest emails.